Core/src/main/java/javapocalypse/core/Application.java

```diff
@@ -0,0 +1,16 @@
+package javapocalypse.core;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.ComponentScan;
+
+@ComponentScan
```

Comment:
I think ComponentScan is not needed and the variable ctx albo is not used so you can just write SpringApplication.run(Application.class). Maybe you have a reason for that ?